src/standalone/Snow/Snow.jl

```diff
@@ -24,16 +54,249 @@ Base.@kwdef struct SnowParameters{FT <: AbstractFloat, PSE}
     θ_r::FT
     "Hydraulic conductivity of wet snow (m/s)"
     Ksat::FT
-    "Critical threshold for snow cover fraction (m)"
-    fS_c::FT
     "Thermal conductivity of ice (W/m/K)"
     κ_ice::FT
-    "Timestep of the model"
-    Δt::Any
+    "Timestep of the model (s)"
```

Comment:
Just curious: why is timestep of the model (s) a parameter of the model?

Reply:
Oh, we compute a physical time to drain liquid water from the snowpack as depth/K, where K is in meters/s and is a conductivity of liquid water in the ice matrix. If that timescale is smaller than the timestep, we use the timestep instead. We could for sure try to improve on this, I agree that it isnt ideal to have physics change with timestep

Reply:
I left a comment!